spring-cloud-dataflow-docs/src/main/asciidoc/configuration.adoc

```diff
@@ -1083,113 +1083,70 @@ being set to `true`.
 
 [[configuration-monitoring-management]]
 == Monitoring and Management
-The Spring Cloud Data Flow server is a Spring Boot application that includes the http://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#production-ready[Actuator
+The Spring Cloud Data Flow server is a Spring Boot 1.5 application that includes the  https://docs.spring.io/spring-boot/docs/1.5.12.RELEASE/reference/htmlsingle/#production-ready[Actuator
```

Comment:
We are making a Spring Boot `1.5.12.RELEASE` reference in 4 places. Shouldn't we use properties for that?

Reply:
we didn't for `current` ;) but yea, generally speaking yes.

Reply:
added as part of https://github.com/spring-cloud/spring-cloud-dataflow/issues/2148